src/Microsoft.CSharp/tests/Microsoft.CSharp.Tests.builds

```diff
@@ -2,7 +2,11 @@
 <Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
   <ItemGroup>
-    <Project Include="Microsoft.CSharp.Tests.csproj" />
+    <Project Include="Microsoft.CSharp.Tests.csproj"/>
```

Comment:
This causes the test project to be built multiple times. Is there a reason we need to do that? Any reason this cannot just be AnyOS and TestTFMs of netcoreapp10,netcore50,net46? This pattern is causing the test build to build much more than necessary on Windows_NT.